cob_cartesian_controller/src/simple_cartesian_interface/simple_cartesian_interface.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/python
+import rospy
+import threading
+import actionlib
+
+from tf.transformations import *
+from geometry_msgs.msg import Pose
+from cob_cartesian_controller.msg import CartesianControllerAction, CartesianControllerGoal
+from cob_cartesian_controller.msg import Profile
+
+
+_transform_listener = None
+_transform_listener_ock = threading.Lock()
+
+_action_client = None
+_action_client_lock = threading.Lock()
+
+def move_lin(pose_goal, frame_id, profile, rotate_only=False):
+    client = actionlib.SimpleActionClient('cartesian_trajectory_action', CartesianControllerAction)
```

Comment:
Yes, I had it our of the function but then thought that this might have been the problem with the crashing server....in the end it was the start_angel=end_angle problem...forgot to change it back...

I'd keep the wait_for_server for re-assuring, the server is still there...